components/ModEditSubmissionForm.vue

```diff
@@ -620,34 +621,46 @@ const validateFields = () => {
     )
 }
 
-function autofillEditSubmissionForm(submissionData: Submission | undefined) {
+function initializeSubmissionFormValues(submissionData: Submission | undefined) {
```

Comment:
This function was updated to create a snapshot of the form before changes are made in submissionBeforeChanges.